### PR TITLE
Set correct repo owner

### PR DESCRIPTION
**Pull request by JesperTerkelsen**
Motivation: Set backstage owner to an active team, to ensure the codebase gets proper maintenance

### CODE REVIEW

File: catalog-info.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: tradeshift-ubl-jsonschema
+  description: |
+    JSON Schema for Universal Business Language
+  annotations:
+    github.com/project-slug: Tradeshift/tradeshift-ubl-jsonschema # GitHub project name
+  tags:
+    - xml
```
Comment (geobos23) [inline catalog-info.yml:14]: shouldn't this be json?

**Review by geobos23 — COMMENTED** [review]
is the tag correct?

File: catalog-info.yml (hunk 1 above)
Comment (JesperTerkelsen) [reply catalog-info.yml:14]: Yeah, will fix

**Review by geobos23 — APPROVED** [review]
:+1:

**Pull request merged**